subsys/bluetooth/host/settings.c

```diff
@@ -180,12 +180,6 @@ static int set(int argc, char **argv, void *value_ctx)
 	return 0;
 }
 
-#if defined(CONFIG_BT_PRIVACY)
-#define ID_SIZE_MAX sizeof(bt_dev.irk)
-#else
-#define ID_SIZE_MAX sizeof(bt_dev.id_addr)
-#endif
```

Comment:
This doesn't seem related, and it was actually removed by one of my own PRs recently.